modules/reference/pages/feature/transportSecurity/examples.adoc

```diff
@@ -57,6 +57,76 @@ You can also run the xref:reference:command/securityUtility-createSSLCertificate
 
 For more information, see xref:reference:config/ssl.adoc[SSL Repertoire].
 
+=== Configure multiple SSL protocols
+
+SSL protocols can be configured by using the `sslProtocol` attribute. Multiple SSL protocols can also be configured on the `sslProtocol` attribute if the underlying JDK support exists. Use a comma-separated list from the TLSv1, TLSv1.1, TLSv1.2, and TLSv1.3 protocols values.
+
+The TLSv1.3 and TLSv1.2 protocols are configured in the following example:
+
+----
+<ssl id="myDefaultSSLConfig"
+      keyStoreRef="defaultKeyStore"
+      trustStoreRef="defaultTrustStore"
+      sslProtocol="TLSv1.3, TLSv1.2" />
+----=== Configure multiple SSL protocols
```

Comment:
I think the edit should end here- looks like the content is duplicated several times after line 71